benchmarks/benchmarks/ag_methods.py

```diff
@@ -0,0 +1,321 @@
+        with pbc active.
+        """
+        self.ag.bbox(pbc=True)
+
+    def time_bbox_no_pbc(self, num_atoms):
+        """Benchmark bounding box calculation
+        with pbc inactive.
+        """
+        self.ag.bbox(pbc=False)
+
+    def time_bsphere_pbc(self, num_atoms):
+        """Benchmark bounding sphere calculation
+        with pbc active.
+        """
+        self.ag.bsphere(pbc=True)
+
+    def time_bsphere_no_pbc(self, num_atoms):
+        """Benchmark bounding sphere calculation
+        with pbc inactive.
+        """
+        self.ag.bsphere(pbc=False)
+
+    def time_center_pbc(self, num_atoms):
+        """Benchmark center calculation with
+        pbc active.
+        """
+        self.ag.center(weights=self.weights,
+                       pbc=True)
+
+    def time_center_no_pbc(self, num_atoms):
+        """Benchmark center calculation with
+        pbc inactive.
+        """
+        self.ag.center(weights=self.weights,
+                       pbc=False)
+
+    def time_centroid_pbc(self, num_atoms):
+        """Benchmark centroid calculation with
+        pbc active.
+        """
+        self.ag.centroid(pbc=True)
+
+    def time_centroid_no_pbc(self, num_atoms):
+        """Benchmark centroid calculation with
+        pbc inactive.
+        """
+        self.ag.centroid(pbc=False)
+
+    def time_concat(self, num_atoms):
+        """Benchmark atomgroup concatenation.
+        """
+        self.ag.concatenate(self.ag)
+
+    def time_difference(self, num_atoms):
+        """Benchmark atomgroup difference
+        operation.
+        """
+        self.ag.difference(self.ag)
+
+    def time_groupby(self, num_atoms):
+        """Benchmark atomgroup groupby
+        operation.
+        """
+        self.ag.groupby('resnames')
+
+    def time_guessbonds(self, num_atoms):
+        """Benchmark atomgroup bond guessing
+        with artificially-seeded vdw values.
+        """
+        self.ag.guess_bonds(self.vdwradii)
+
+    def time_intersection(self, num_atoms):
+        """Benchmark ag intersection.
+        """
+        self.ag.intersection(self.ag)
+
+    def time_strict_subset(self, num_atoms):
+        """Benchmark ag strict subset operation.
+        """
+        self.ag.is_strict_subset(self.ag)
+
+    def time_strict_superset(self, num_atoms):
+        """Benchmark ag strict superset operation.
+        """
+        self.ag.is_strict_superset(self.ag)
+
+    def time_disjoint(self, num_atoms):
+        """Benchmark disjoint operation between
+        atomgroups.
+        """
+        self.ag.isdisjoint(self.ag)
+
+    def time_subset(self, num_atoms):
+        """Benchmark subset operation between
+        atomgroups.
+        """
+        self.ag.issubset(self.ag)
+
+    def time_superset(self, num_atoms):
+        """Benchmark superset operation between
+        atomgroups.
+        """
+        self.ag.issuperset(self.ag)
+
+    def time_packing(self, num_atoms):
```

Comment:
I assume this should be called `time_pack_into_box` to make clearer what is benchmarked?

Reply:
I'd suggest to use the same name as the method